schemas/test/en/test_list_collector_variants_section_summary_items.json

```diff
@@ -0,0 +1,538 @@
+{
```

Comment:
I think we should rename the existing schema from `test_list_collection_section_summary` to `test_list_collection_list_summary`. 
Then yours can drop the `_items` bit of the name.

Reply:
Schemas renamed now, thanks!